src/com/googlecode/shavenmaven/Dependencies.java

```diff
@@ -49,11 +49,11 @@ public boolean update(File directory) {
     }
 
     public static void main(String[] args) throws Exception {
-        if (args.length == 0 || args.length > 2) {
```

Comment:
This commit was just laying the groundwork for later splitting command line arguments between options & args.  I'm happy to squash this commit into that later one if you prefer a cleaner/shorter commit history.